src/NativeScript/ObjC/Unmanaged/UnmanagedPrototype.h

```diff
@@ -0,0 +1,33 @@
+#ifndef UnmanagedPrototype_h
+#define UnmanagedPrototype_h
+
+namespace NativeScript {
+class UnmanagedPrototype : public JSC::JSDestructibleObject {
```

Comment:
`UnmanagedPrototype` doesn't need destruction, so it can be a plain old `JSNonFinalObject`.